app/controllers/employees_controller.rb

```diff
@@ -20,25 +20,32 @@ def create
   end
   
   def update
-    employee = Employee.find(params[:id])
-
-    if employee.update(employee_params)
-      render json: employee
+  
+    if  @employee.update(employee_params)
+      render json:  @employee
     else
-      render json: employee.errors, status: :unprocessable_entity
+      render json:  @employee.errors, status: :unprocessable_entity
     end
   end
 
   def destroy 
-    employee = Employee.find(params[:id])
-    employee.destroy
+    @employee.destroy
```

Comment:
Use if else condition if destroy then send status message or else send the error as we are sending in create action